lib/sda/host_msg.go

```diff
@@ -61,10 +67,13 @@ type TokenID uuid.UUID
 type Token struct {
 	EntityListID EntityListID
 	TreeID       TreeID
-	ProtoID      ProtocolID
-	RoundID      RoundID
-	TreeNodeID   TreeNodeID
-	cacheId      TokenID
+	// TO BE REMOVED
```

Comment:
Ok since we still can instantiate a protocol instance directly (our whole test-code base does that), ProtocolID is kept.